app/controllers/lti_controller.rb

```diff
@@ -0,0 +1,81 @@
+class LtiController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: :launch # for lti integration
+  before_action :set_group_assignment, only: %i[launch]
+  before_action :set_lti_params, only: %i[launch]
+  after_action :allow_iframe_lti, only: %i[launch]
+  
+  def launch
+    session[:isLTI]=true # the lti session starting
+    require 'oauth/request_proxy/action_controller_request'
```

Comment:
Why was this needed ?

Reply:
mistakenly added now removed

Reply:
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/52851184/122669039-0233f500-d1d9-11eb-8195-64ee32c12c82.png">
else this error is happening if I remove this

Reply:
Then move this to top of the file

Reply:
done